src/Plugins/LockingMiddleware.php

```diff
@@ -23,9 +23,7 @@ class LockingMiddleware implements Middleware
     /**
      * Queues incoming commands until the first has completed
      *
-     * @param object $command
-     * @param callable $next
-     * @return mixed
+     * {@inheritdoc}
```

Comment:
Not now, but as @rosstuck said, he will improve docs when he has time. Also, @mvriel pointed that there will be an official inheritDoc tag, which I think will be used in the league (as the curly-braced version is pretty common nowadays).